app/assets/javascripts/listings/ListingEligibilityService.js.coffee

```diff
@@ -36,19 +36,19 @@ ListingEligibilityService = ($localStorage, ListingIdentityService, ListingUnitS
     minMax = [1, 1]
     if listing.unitSummary
       min = _.min(_.map(listing.unitSummary, 'minOccupancy')) || 1
-      max = _.max(_.map(listing.unitSummary, 'maxOccupancy')) || Service.defaultMaxOccupancy(amiLevel)
+      max = _.max(_.map(listing.unitSummary, 'maxOccupancy')) || Service.maxNumOfHousehold(amiLevel)
       minMax = [min, max]
     return minMax
 
-  Service.defaultMaxOccupancy = (amiLevel) ->
+  Service.maxNumOfHousehold = (amiLevel) ->
     _.max(_.map(amiLevel.values, (v) -> v.numOfHousehold))
 
   Service.occupancyIncomeLevels = (listing, amiLevel) ->
     return [] unless amiLevel
     occupancyMinMax = Service.occupancyMinMax(listing, amiLevel)
     min = occupancyMinMax[0]
-    # We add '+ 2' for 2 children under 6 as part of householdsize but not occupancy
-    max = occupancyMinMax[1] + 2
+    # We add '+ 2' for 2 children under 6 as part of householdsize but not occupancy. Unless it's max
```

Comment:
it seems like these two functions are both taking different weird responsibilities for figuring out this number. I wonder if occupancyMinMax should just return null if maxOccupancy is null, then we can deal with the logic of figuring out the max based on the AMI level and whether or not to add 2 in this function